Runner/Utils.py

```diff
@@ -0,0 +1,300 @@
+import azure.storage.blob as azureblob
+import azure.batch.models as batchmodels
+import datetime
+import time
+import os
+from enum import Enum
+from xml.etree.ElementTree import Element, SubElement, tostring, ElementTree
+from pytz import timezone
+import pytz
+utc = pytz.utc
+import logging
+
+logger = logging.getLogger('rendering-log')
+logger.setLevel(logging.DEBUG)
+# create file handler which logs even debug messages
+fh = logging.FileHandler('template.log')
+fh.setLevel(logging.DEBUG)
+# create console handler with a higher log level
+ch = logging.StreamHandler()
```

Comment:
:bulb: **Recommendation** rename to `console_handler` or something more obvious.

Reply:
okay, i'll also do fh as well